src/code/z_en_hy_code.c

```diff
@@ -69,13 +69,13 @@ EnDoor* EnHy_FindNearestDoor(Actor* actor, PlayState* play) {
     do {
         doorIter = SubS_FindActor(play, doorIter, ACTORCAT_DOOR, ACTOR_EN_DOOR);
         door = (EnDoor*)doorIter;
-        dist = Actor_WorldDistXYZToActor(actor, &door->dyna.actor);
+        dist = Actor_WorldDistXYZToActor(actor, &door->door.dyna.actor);
```

Comment:
I don't like the look of this `door->door` access. One way to fix it would be to change the name of the `door` variable here, but this pops up in a lot of different places too, and changing them all might be annoying. The root cause is because EnDoor has a substruct in it called `door`:
```c
   /* 0x000 */ KnobDoorActor door;
```

What do you think about changing that to `KnobDoorActor knobDoor`? Or something similar to that. Would that cause problems somewhere else? That would eliminate this `door->door` access and also make it more clear which struct is being accessed here.

Reply:
Oh, good idea, I'll try to fiddle with it a bit

Reply:
I applied this idea here: https://github.com/zeldaret/mm/pull/1227/commits/b537b75f061cd36bc3a05f6fb44788ed47519374

I also renamed the `SlidingDoorActor` member of some actors from `door` to `slidingDoor` to avoid the same `door->door` issue.
lmk what you think

Reply:
Looks great to me, thanks for doing it!